website/data/pages/radius-&-shadows.mdx

```diff
@@ -1,3 +1,34 @@
 ---
 title: Radius & shadows
----
\ No newline at end of file
+description:
+---
+
+# Radius & shadows
+
+## Border Radius
+
+To optimize the global consistency, all elements with a corner radius are standardized at 4px.
+
+![Border radius](/border-radius.png)
+## Examples:
+**‣**  Buttons <br/> <br/>
+**‣**  Filters <br/> <br/>
+**‣**  Tables <br/> <br/>
+**‣**  Content container <br/> <br/>
+**‣**  Dropdown <br/> <br/>
+
+## Drop shadows
+
+In order to keep consistency across the product, the shadows also follow certain rules.
+
+![Drop shadows](/drop-shadows.png)
+We use it on 3 differents cases : <br/> <br/>
+**‣**  On the content containers (in addition with the border below) <br/>
+**‣**  On the Filters <br/> 
+**‣**  On the PopOvers & Modals <br/> 
```

Comment:
Popover not PopOver

Reply:
it's spelled as PopOver in the [UI Kit in Figma](https://www.figma.com/file/PICiE8O4NLrHO1lhJftLdE/Strapi---UI-Kit-%F0%9F%A7%A9?node-id=4183%3A64018). @maevalienard what is the correct variant ?
![image](https://user-images.githubusercontent.com/47764205/143623591-b9819000-a837-4f30-8e5b-3d43345fa7c4.png)

Reply:
It's spelled as PopOver in the [UI Kit in Figma](https://www.figma.com/file/PICiE8O4NLrHO1lhJftLdE/Strapi---UI-Kit-%F0%9F%A7%A9?node-id=4183%3A64018). @maevalienard what is the correct variant ?
![image](https://user-images.githubusercontent.com/47764205/143623591-b9819000-a837-4f30-8e5b-3d43345fa7c4.png)

Reply:
I usually write it PopOvers, but it doesn't matter that much in that context, both works IMO :)